### ISSUE TITLE
Release yesno 0.1.3

### ISSUE THREAD

**Issue by aylapear** [poissonconsulting/yesno#11]
Prepare for release:

* [x] `git pull`
* [x] Check [current CRAN check results](https://cran.rstudio.org/web/checks/check_results_yesno.html)
* [x] [Polish NEWS](https://style.tidyverse.org/news.html#news-release)
* [x] `urlchecker::url_check()`
* [x] `devtools::build_readme()`
* [x] `devtools::check(remote = TRUE, manual = TRUE)`
* [x] `devtools::check_win_devel()`
* [x] `revdepcheck::revdep_check(num_workers = 4)`
* [x] Update `cran-comments.md`
* [x] `git push`

Submit to CRAN:

* [x] `usethis::use_version('patch')`
* [x] `devtools::submit_cran()`
* [x] Approve email

Wait for CRAN...

* [x] Accepted :tada:
* [x] `usethis::use_github_release()`
* [x] `usethis::use_dev_version(push = TRUE)`

**Comment by aylapear** [comment]
```
devtools::check(remote = TRUE, manual = TRUE)
```

Output this note

```
❯ checking HTML version of manual ... NOTE
  Found the following HTML validation problems:
  yesno.html:15:44 (yesno.Rd:5): Error: <main> is not recognized!
  yesno.html:15:44 (yesno.Rd:5): Warning: discarding unexpected <main>
  yesno.html:89:1 (yesno.Rd:44): Warning: discarding unexpected </main>
  yesno.html:4:1 (yesno.Rd:5): Warning: <link> inserting "type" attribute
  yesno.html:12:1 (yesno.Rd:5): Warning: <script> proprietary attribute "onload"
  yesno.html:12:1 (yesno.Rd:5): Warning: <script> inserting "type" attribute
  yesno.html:17:1 (yesno.Rd:5): Warning: <table> lacks "summary" attribute
  yesno.html:38:1 (yesno.Rd:10): Warning: <table> lacks "summary" attribute
  yesno.html:39:14 (yesno.Rd:10): Warning: <code> attribute "id" has invalid value "..."
  yesno2.html:15:44 (yesno2.Rd:5): Error: <main> is not recognized!
  yesno2.html:15:44 (yesno2.Rd:5): Warning: discarding unexpected <main>
  yesno2.html:85:1 (yesno2.Rd:36): Warning: discarding unexpected </main>
  yesno2.html:4:1 (yesno2.Rd:5): Warning: <link> inserting "type" attribute
  yesno2.html:12:1 (yesno2.Rd:5): Warning: <script> proprietary attribute "onload"
  yesno2.html:12:1 (yesno2.Rd:5): Warning: <script> inserting "type" attribute
  yesno2.html:17:1 (yesno2.Rd:5): Warning: <table> lacks "summary" attribute
  yesno2.html:38:1 (yesno2.Rd:10): Warning: <table> lacks "summary" attribute
  yesno2.html:39:14 (yesno2.Rd:10): Warning: <code> attribute "id" has invalid value "..."
```

**Comment by aylapear** [comment]
Fixed issue by running `brew install tidy-html5`

**Comment by aylapear** [comment]
```
> revdepcheck::revdep_check(num_workers = 4)
── INIT ──────────────────────────────────────────────────────────────────────────────────────────── Computing revdeps ──
── INSTALL ──────────────────────────────────────────────────────────────────────────────────────────────── 2 versions ──
Installing CRAN version of yesno
Installing DEV version of yesno
── CHECK ────────────────────────────────────────────────────────────────────────────────────────────────── 8 packages ──
✔ jdenticon 0.1.1                        ── E: 0     | W: 0     | N: 0                                                   
✔ available 1.1.0                        ── E: 0     | W: 0     | N: 0                                                   
✔ greta 0.4.5                            ── E: 1     | W: 0     | N: 0                                                   
✔ rjtools 1.0.13                         ── E: 0     | W: 0     | N: 0                                                   
✔ settingsSync 3.0.2                     ── E: 0     | W: 0     | N: 0                                                   
✔ sims 0.0.3                             ── E: 1     | W: 0     | N: 0                                                   
✔ labNorm 1.0.1                          ── E: 0     | W: 0     | N: 1                                                   
✔ ypr 0.6.0                              ── E: 0     | W: 0     | N: 0                                                   
OK: 8                                                                                                                  
BROKEN: 0
Total time: 6 min
── REPORT ───────────────────────────────────────────────────────────────────────────────────────────────────────────────
Writing summary to 'revdep/README.md'
Writing problems to 'revdep/problems.md'
Writing failures to 'revdep/failures.md'
Writing CRAN report to 'revdep/cran.md'
```

**Comment by aylapear** [comment]
New version was successfully accepted by CRAN
